CorsixTH/Lua/epidemic.lua

```diff
@@ -112,6 +112,7 @@ function Epidemic:tick()
       self:checkNoInfectedPlayerHasLeft()
       self:markedPatientsCallForVaccination()
       self:showAppropriateAdviceMessages()
+      self:checkThereAreNoAnyInfectedPatientsLeft()
```

Comment:
Potentially sanity check the order of these, I'm sure it's still fine.

Reply:
> Potentially sanity check the order of these, I'm sure it's still fine.

I changed the order, but honestly I don't think it matters here. but thanks for the tip.